src/App/App.js

```diff
@@ -5,38 +5,41 @@ import Header from '../Header/Header';
 import Modal from '../Modal/Modal';
 // import RabbitTrail from '../RabbitTrail/RabbitTrail'
 import CardContainer from '../CardContainer/CardContainer';
+import {getAllMovies, getMovie } from '../util.js';
 import movieData from '../movieData';
 
 class App extends Component {
   constructor() {
     super();
     this.state = {
-      movieData: movieData.movies,
+      movieData: [],
```

Comment:
Now if `this.state.movieData.length` is falsy, we will be able to show a certain error message.